packages/component-submission/server/resolvers/submitManuscript.js

```diff
@@ -31,7 +31,7 @@ async function submitManuscript(_, { data }, { user, ip }) {
 
   manuscript.applyInput(manuscriptInput)
 
-  const sourceFile = manuscript.getSource()
+  const sourceFile = await manuscript.getSource()
```

Comment:
This needs awaiting - just this change seemed to fix some of the unit tests failing